nicoleWojewoda/assignments/treesBFS/lc102/lc102.js

```diff
@@ -0,0 +1,113 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent its level-by-level traversal. You should populate the values of all nodes of each level from left to right in separate sub-arrays.
```

Comment:
Nice overview here. Would also suggest writing some additional test cases, especially for an edge case.